### ISSUE TITLE
[DOC] Unclear origin of generate-jwt-hs256-server-token command

### ISSUE THREAD

**Issue by brainstorm** [mozilla/sccache#771]
While going step by step through the distributed quickstart:

https://github.com/mozilla/sccache/blob/master/docs/DistributedQuickstart.md#configure-a-build-server

The comment under `scheduler_auth` states:

> \# This will be generated by the `generate-jwt-hs256-server-token` command or
> \# provided by an administrator of the sccache cluster.
> 

Where is that (sub)command?

**Comment by brainstorm** [comment]
I just noticed it is mentioned and described in the other distributed document:

https://github.com/mozilla/sccache/blob/master/docs/Distributed.md#jwt-hs256-preferred

It should be cross-referenced.

**Comment by anholt** [comment]
Yeah, it would have really helped me to have those two docs merged, with "Installing", "Setting up your central scheduler", "Setting up a build server against a given scheduler" and "Connecting your client" as the sections.

(Also some demo of how to set up https if we want this to still be quickstart :) )
